src/stores/video.ts

```diff
@@ -279,6 +279,30 @@ export const useVideoStore = defineStore('video', () => {
     }
   }
 
+  // Used to download a file from the video recovery database
+  const downloadTempVideoDB = async (): Promise<void> => {
+    console.debug('Downloading video chunks from the temporary database.')
+    const zipWriter = new ZipWriter(new BlobWriter('application/zip'))
+
+    const fileNames = await tempVideoChunksDB.keys()
+
+    const filesPromises = fileNames
+      .filter(async (filename) => await tempVideoChunksDB.getItem(filename))
```

Comment:
I'm not sure if this filter is needed here because I think filter does not support async operations and will just return all elements as a simple map

Reply:
That probably need to be overwrite with `Promise.allSettled(fileNames.map...).filter`

Reply:
Didn't get it. The two approaches are the same.

Reply:
Talked with @JoaoMario109 and got the problem. Will change.